R/fd_belowground.R

```diff
@@ -8,6 +8,8 @@
 #' @author Measurements by Kayla Mathes at the University of Michigan Biological Station.
 #' @examples
 #' fd_soil_respiration()
+library(lubridate)
```

Comment:
In package code, `library` calls should never appear in the functions; any required or suggested packages should be added to the DESCRIPTION file. See http://r-pkgs.had.co.nz/description.html#dependencies

Reply:
`lubridate` is removed and used `as.Date()` instead. This was better than `strptime()` because it doesn't add time. Passed checks